content/dart/concepts/list/terms/join/join.md

```diff
@@ -0,0 +1,108 @@
+---
+Title: 'join' 

```

Comment:
```suggestion
Title: '.join()' 

```

Reply:
How about this fragment, it says differently?
![metadata](https://github.com/Codecademy/docs/assets/157640220/9360a104-7d74-4156-b95b-8221c40594e0)